crates/ruff_linter/src/rules/refurb/rules/reimplemented_starmap.rs

```diff
@@ -231,7 +237,7 @@ impl StarmapCandidate<'_> {
     /// Try to produce a fix suggestion transforming this node into a call to `starmap`.
     pub(crate) fn try_make_suggestion(
         &self,
-        name: String,
+        name: Name,
         iter: &Expr,
```

Comment:
Do you think it would be useful to use `Into<Name>` in the functions like this? That way one can directly pass in `String` / `&str` and not worry about constructing the `Name`. If it's too much work (I guess it is?), it's fine not to do it.

Reply:
Not sure. I'm somewhat concerned about accidental monomorphization because some of the methods aren't trivial.

Reply:
It's fine to not do it, just wondered if it would be useful.